installFeature task fails after version bump to 3.3 plugin with gradle 7.2

No issues with 3.2.1. After version bump to 3.3:
```
> Task :installFeature FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Some problems were found with the configuration of task ':installFeature' (type 'InstallFeatureTask').
  - In plugin 'liberty' type 'io.openliberty.tools.gradle.tasks.InstallFeatureTask' property 'additionalJsonList' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#missing_annotation for more details about this problem.
  - In plugin 'liberty' type 'io.openliberty.tools.gradle.tasks.InstallFeatureTask' property 'newProject' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#missing_annotation for more details about this problem.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 11s
8 actionable tasks: 8 executed
```

--- a/src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractFeatureTask.groovy
+++ b/src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractFeatureTask.groovy
@@ -22,6 +22,7 @@ import org.gradle.api.logging.LogLevel
 import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.options.Option
+import org.gradle.api.tasks.Input
 import org.gradle.api.Project
 import org.gradle.testfixtures.ProjectBuilder
 
@@ -40,6 +41,7 @@ public class AbstractFeatureTask extends AbstractServerTask {
 
     private InstallFeatureUtil util;
 	
+    @Internal
 	Project newProject = project;
 
     @Option(option = 'serverDir', description = '(Optional) Server directory to get the list of features from.')
@@ -98,6 +100,7 @@ public class AbstractFeatureTask extends AbstractServerTask {
         public File downloadArtifact(String groupId, String artifactId, String type, String version) throws PluginExecutionException {
 			
 			String coordinates = groupId + ":" + artifactId + ":" + version + "@" + type
+
 			def dep = newProject.dependencies.create(coordinates)
 			def config = newProject.configurations.detachedConfiguration(dep)
 	
@@ -143,6 +146,7 @@ public class AbstractFeatureTask extends AbstractServerTask {
         return features
     }
 	
+	@Internal
 	protected List<String> getAdditionalJsonList() {
 		List<String> result = new ArrayList<String>()
 		project.configurations.featuresBom.dependencies.each { dep ->
@@ -205,7 +209,7 @@ public class AbstractFeatureTask extends AbstractServerTask {
     }
 
     protected InstallFeatureUtil getInstallFeatureUtil(Set<String> pluginListedEsas, List<ProductProperties> propertiesList, String openLibertyVerion, String containerName, List<String> additionalJsons) throws PluginExecutionException {
-		//if installing userFeature, recompile gradle to find mavenLocal artifacts created by prepareFeature task. 
+		//if installing userFeature, recompile gradle to find mavenLocal artifacts created by prepareFeature task.
 		if(project.configurations.featuresBom.dependencies) {  
 			try {
 				ProjectBuilder builder = ProjectBuilder.builder();

--- a/src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractPrepareTask.groovy
+++ b/src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractPrepareTask.groovy
@@ -22,6 +22,7 @@ import org.gradle.api.artifacts.ResolveException
 import org.gradle.api.logging.LogLevel
 import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.options.Option
+import org.gradle.api.tasks.Internal
 
 import io.openliberty.tools.common.plugins.util.PrepareFeatureUtil
 import io.openliberty.tools.common.plugins.util.PluginExecutionException
@@ -114,6 +115,7 @@ public class AbstractPrepareTask extends AbstractServerTask {
         }
     }
 	
+	@Internal
 	protected List<String> getDependencyBoms() {
 		List<String> result = new ArrayList<String>()
 		project.configurations.featuresBom.dependencies.each { dep ->

